docs/architecture/ADR-002-QGB-Overview.md

```diff
@@ -0,0 +1,47 @@
+# ADR 003: ValSet
+
+## Changelog
+
+- {date}: {changelog}
+
+## Context
+Various Rollups on EVM L1s may have a need for using Celestia as a DA layer, while keeping fraud/validity proof verification in the EVM.
+To that end, two concrete possibilities exist: the Gravity bridge and a native EVM light client.
+
+## Approaches
+The following issue summarizes the two viable approaches to accommodate this: [#4](https://github.com/celestiaorg/quantum-gravity-bridge/issues/4)
+
+## Decision
+We decided to go for the gravity bridge approach and move the orchestrator/relayer logic to Celestia-app.
+
+## Detailed Design
+The QGB allows Celestia block header data roots to be relayed in one direction, from Celestia to an EVM chain. It does not support
+bridging assets such as fungible or non-fungible tokens directly, and cannot send messages from the EVM chain back to Celestia.
```

Comment:
Don't manually add newlines for markdown files, except maybe after sentences if you want each sentence on a different line for git diffs.
```suggestion
The QGB allows Celestia block header data roots to be relayed in one direction, from Celestia to an EVM chain. It does not support bridging assets such as fungible or non-fungible tokens directly, and cannot send messages from the EVM chain back to Celestia.
```